README.md

```diff
@@ -35,7 +60,15 @@ class PostsController < ApplicationController
 end
 ```
 
-3. **(optional)** override custom cache key data. For default, cache key is defined by URL and query string
+5. **(optional)** set a custom TTL for the cache by overriding the `write_to_backing_cache_store` method in your initializer file
+```ruby
+CACHE_TTL = 30.minutes
```

Comment:
Let's make it clear that this method and constant needs to be defined inside `module ResponseBank`.